### ISSUE TITLE
false positive when indenting multiline strings

### ISSUE THREAD

**Issue by bukzor** [PyCQA/pycodestyle#265]
``` python
from textwrap import dedent

print dedent(
    '''
        rm -rf ./build/.git*
        rm -rf ./{build}/%(revision)s/
        mkdir -p ./{build}/
        mv ./build/ ./{build}/%(revision)s/
    '''.format(
        build='build',
        a=1,
        b=2,
        # more stuff
    )
)
```

yields these errors:

```
scratch.py|24 col 9 error| continuation line over-indented for hanging indent [E126] [python/flake8]
scratch.py|28 col 5 error| continuation line under-indented for hanging indent [E121] [python/flake8]
```

To get pep8 happy, I have to indent like so:

``` python
from textwrap import dedent

print dedent(
    '''
        rm -rf ./build/.git*
        rm -rf ./{build}/%(revision)s/
        mkdir -p ./{build}/
        mv ./build/ ./{build}/%(revision)s/
    '''.format(
    build='build',
    a=1,
    b=2,
    # more stuff
)
)
```
